src/components/NetworkSwitcher/NetworkSwitcher.component.tsx

```diff
@@ -91,3 +112,23 @@ const EthereumOptionPopover = ({ children, show }: EthereumOptionPopoverProps) =
     </StyledPopover>
   )
 }
+
+const WrongNetworkMobileModal = () => {
+  return (
+    <Modal isOpen={true} onDismiss={() => null} maxHeight={90}>
```

Comment:
`isOpen` without assignment `={true}` will work too as it's a shorthand syntax when prop is a boolean.